Tests/ConsoleKitTests/Utilities.swift

```diff
@@ -47,22 +47,23 @@ final class SubGroup: CommandGroup {
 }
 
 final class TestCommand: Command {
-    let arguments: [CommandArgument] = [
-        .argument(
-            name: "foo",
-            help: ["A foo is required", "An error will occur if none exists"]
-        )
-    ]
-
-    let options: [CommandOption] = [
-        .value(name: "bar", short: "b", help: ["Add a bar if you so desire", "Try passing it"])
-    ]
+    struct Signature: Inputs {
+        let foo = Argument<String>(name: "foo", help: """
+        A foo is required
+        An error will occur if none exists
+        """)
+        
+        let bar = Option<String>.value(name: "bar", short: "b", help: """
```

Comment:
It looks kind of awkward having `Option` use a static func for initialization here alongside `Argument` which uses a normal one. Maybe we should have `Option` have two separate init methods with different argument labels.